zk_toolbox/crates/zk_inception/src/forge_utils.rs

```diff
@@ -12,3 +13,21 @@ pub fn fill_forge_private_key(
     }
     Ok(forge)
 }
+
+pub async fn check_the_balance(forge: &ForgeScript) -> anyhow::Result<()> {
+    let mut not_enough_money = true;
+    while not_enough_money {
```

Comment:
It seems we don't really need a while loop here.

Reply:
is there any other way, how to check it every time? 
we have two ways to close the loop, have enough money of forcefully continue